### PR TITLE
Increment version number and CHANGELOG to close Milestone 2

### PR DESCRIPTION
**Pull request by stratofax**
Not much going on here but this is what it looks like when I increment the version number.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,17 @@
 
 A version history of BallotLab.
 
-## 0.1.2 - September 7, 2022
+## 0.1.3 -- September 20, 2022
```
Comment (ion-oset) [inline CHANGELOG.md:5]:
```suggestion
## 0.1.3 - September 20, 2022
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,17 @@
...
+
+Complete [2. Generate ballot PDFs with embedded form objects Milestone](https://github.com/TrustTheVote-Project/BallotLab/milestone/2)
```
Comment (ion-oset) [inline CHANGELOG.md:7]:
```suggestion
Completed [Milestone 2. Generate ballot PDFs with embedded form objects Milestone](https://github.com/TrustTheVote-Project/BallotLab/milestone/2)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,7 +2,17 @@
...
+
+* Prototype check boxes to track coordinates of vote marking ovals
+* Generate ballots with hidden, embedded form objects to track coordinates and ids of vote mark ovals
+* Generate ballots from all four (4) precincts in the September 2022 EDF
+* Incorporate new NISTLib data access layer (includes corresponding test harness)
+* Additional layout enhancements and refinements
+
+## 0.1.2 -- September 7, 2022
 
 Complete Milestone 1: flat file PDF. Notable features include:
```
Comment (ion-oset) [inline CHANGELOG.md:17]:
```suggestion
Completed milestone 1: [Generate flat PDFs from Jetsons EDF](https://github.com/TrustTheVote-Project/BallotLab/milestone/1)
```

**Review by ion-oset — CHANGES_REQUESTED** [review]

**Comment by stratofax** [comment]
@ion-oset I believe I made all the changes you requested, and also removed a considerable amount of dead or unneeded code. Please take a last look and then I'll merge this work onto `development` and then onto main.

Thanks again for your excellent reviews!

**Review by ion-oset — APPROVED** [review]
LGTM :+1:

I think this PR warrants a squash merge.

**Comment by stratofax** [comment]
Agreed: squash merge.

**Pull request merged**